services/swagger.js

```diff
@@ -21,7 +21,7 @@ var httpMethods = ['get', 'put', 'post', 'delete', 'options', 'head', 'patch'];
 
 exports.init = function(logger, config, callback) {
     var cfg = config.get('swagger');
-    var responseModelValidationLevel = /error|warn/.test(cfg.validateResponseModels) ? cfg.validateResponseModels : 0;
+    var responseModelValidationLevel = /error|warn|fail/.test(cfg.validateResponseModels) ? cfg.validateResponseModels : 0;
```

Comment:
do something to make this calculated value available as a property of the service
(probably want to calculate the polymorphic validation value here too)